### ISSUE TITLE
Enhance `reanalysis-forced` notebook

### ISSUE THREAD

**Issue by navidcy** [COSIMA/regional-mom6#96]
Let's enhance the notebook `reanalysis-forced` at https://regional-mom6.readthedocs.io/en/latest/demo_notebooks/reanalysis-forced.html with some plots of, e.g, the domain or the initial condition etc.

Also let's add a thumbnail to appear at https://regional-mom6.readthedocs.io/en/latest/demos.html

**Comment by angus-g** [comment]
Maybe moving discussion from #99 here: I've implemented a way to run the notebooks in the container on GitHub (where the model, tools, and input data are available), and provide that notebook to RTD. Example build is https://regional-mom6.readthedocs.io/en/angus-g-render-notebooks/demo_notebooks/reanalysis-forced.html, from the angus-g/render-notebooks branch.

At the moment, there's both the GitHub push webhook, and the RTDS action webhook firing for the same version, so it's not perfectly integrated until we completely switch over. There might be a little bit more to do in generating previews for PRs (for this demo, I had to manually activate the branch on RTD).

**Comment by navidcy** [comment]
This looks great!

**Comment by aekiss** [comment]
> Also let's add a thumbnail to appear at https://regional-mom6.readthedocs.io/en/latest/demos.html

thumbnail still missing

**Comment by navidcy** [comment]
Yes. Related to this issue is https://github.com/COSIMA/regional-mom6/issues/119 which if resolved then we could have the notebook cells evaluated locally and push.
